local quotes = {
	"Bro is working at sy bau ğŸ˜­ğŸ˜­",
	"someone server side the game",
	"broâ– your larping knowledge",
	"Your safety matters. Not feeling comfortable? You can block or report this person anytime from their profile.",
	"Textcode.dll STILL NOT DETECTED, BYPASSES HYPERION âœ…âœ…âœ… NO AMDXX64.DLL âœ…âœ…âœ…",
	"free btool: replicatesignal(game.Players.LocalPlayer.Character.Humanoid.ServerEquipTool, workspace.Baseplate)",
	"where is a serverside script that is compatible with delta and it shows on the cloud icon and does workâ– I can't find any serverside that works",
	"I'm a Jerk who's trying to Lag the creator of my exploit to death.",
	"dude u reallise i can shut down the server",
	"TextBox",
	"Ok Exspliot dude â˜ ",
	"I brb ima equip a fe",
	"METABLES ISNT A THINGS",
	"dude i use a 10.00 exec",
	"do you even have infinite yield",
	--"can somekne help me get hac",
	"do u have a \"to\" cmd?",
	"why are no esp or aimbot loadstrings working:?",
	"infinite yield is not working in this game i think",
	"Gooner game ğŸ’€ğŸ’€ğŸ’€ğŸ’€",
	"GFJIJK YF ABU",
	"free team: workspace.Remote.loadchar:InvokeServer(ply, \"Bright yellow\")",
	"Bro game.Players is better than game:GetService\"Players\"",
	"Imagine expolaiting",
	"How To Make A ROBLOX Executor | LEVEL 7 EASY PASTEBIN",
	"Textcode.dll.dll successfully injected!",
	"imagine hacking because your life sucks",
	"Nyx leaked 2015",
	"?crasj",
	"Protosmasher UI Rewrite found leaked",
	"nathan chay reference",
	"omg ima co-owner",
	"if ur co owner fix the haks",
	"/give Fe bypass By code 29292",
	"The devs patched the spin command",
	"Dawg yall are jus copying what i said",
	"roblox rcc leak 2025",
	"yea i jerk it to miku so what",
	"elysian on top austin still updates it",
	"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible.",
	"Roblox.Thumbs.Dispatcher.get_Singleton() +0",
	"Roblox.Global..cctor() +173",
	"Roblox.Thumbs.Properties.Settings.get_ThumbnailerEnabled() +23",
	"Account State override: None Remind Warn Delete Poison",
	"bro got banned on ####mate ranked im crying",
	"Define Print Lil Bro ğŸ’€ğŸ’€ğŸ’€ğŸ’€",
	"Define game:GetService Lil Bro ğŸ’€ğŸ’€ğŸ’€",
	"Define getgenv Lil Bro ğŸ’€ğŸ’€ğŸ’€ğŸ’€",
	"do you use bloxstrap",
	"it refers to the client side player ####",
	"bruh y'all are hackers im reporting all of you",
	"?sbawn car",
	"Did y'all ğŸ‘‚ğŸ‘‚ğŸ‘‚ that?",
	"?kill all nuke 30 sekend",
	"This green apple dude",
	"sbas i am no haker",
	"?kick maks",
	"I will report him million times",
	"WARNING!!! Aesthetical HAS LAUNCHED THE DEATHNUKE, EVERYONE WILL DIE IN T MINUS 3 SECONDS(S)!!!",
	"Do NOT execute his script it uses BrowserService ExecuteJavaScript which can be used to log your roblox account !! Stay safe guys Dont Execute Any scripts from that guy !!!",
	"You will be banned",
	"Reporting you",
	"hackers",
	"hackers report",
	"Yes",
	"But don't if they kill cops",
	"shut up",
	"!crawl",
	"Ok i shutdown this server",
	"Team now.",
	"anyone got 99:00 AM as they're time?",
	"isnt task.wait fps based too",
	"game.ServerScriptService.CellManager â˜ â˜ â˜ â˜  Those who know",
	"Closed again! WARNING: This is far from the final version. A lot of features are missing, and some features may be removed(Like crafting). This beta preview is to test for lag, bugs, and glitches.â– Also, don't get your hopes up that this version will be released soon. It probably won't.",
	"ye using exploits cuz i believe in greek gods",
	"reporting to roblox is like telling this dude to go outside",
	"Now if who's try arest I'm gona arested peacose I'm not in jail i don't fast",
	"1v1 plt kid",
	"do you guys code",
	"Rise and shine, Mister Robloxian. Rise and... shine. Not that I... wish to imply you have been sleeping on the job.â– No one is more deserving of a rest, and all the effort in the world would have gone to waste until... well, let's just say your hour has... come again.â– The right man in the wrong place can make all the difference in the world. So, wake up, Mister Robloxian. Wake up and... *smell the ashes*...",
	"stop exoloiting.",
	"What is a level 10 delta",
	"CRASHING SERVER!! Script made by jake11price.",
	"my xbox360 blow out what do i doâ– By the rivers of Babylon, there we sat down, yea, we wept, when we remembered Zion. Remember, O Lord, The Children of Edom in the day of Jerusalem who said, \"Raze it, raze it, even to the foundation.\"",
	"Same account launched game from different device. Reconnect if you prefer to use this device.",
	"N####kernel tapped in watch out im level 9 + 10",
	"ur user id btw",
	"wait lemmeee pull up my haccss\\",
	"bro why do you even care",
	[[Ø±Ø³Ø§Ù„Ø© Ù…Ù† Ø§Ù„Ù„Ù‡ ØªØ¹Ø§Ù„Ù‰ roblox free tostring synapse v3rmillion fe harkinian bypass harked invisible fe elysian jjsploit Ø®Ø¯Ø§Ø¹ Ø£ÙŠ ÙÙŠØ±ÙˆØ³ ØªØ­Ù…ÙŠÙ„ Ù…Ø¬Ø§Ù†ÙŠ]],
	"how the #### am i suposed to make my own gui i legit suck",
	"i am not a kid i am 42",
	"who got pushed down the stairs...",
	"A lot changes in 4 years.",
	"Hatsune Miku died on a leek for you, defend her death, she defeated your sins, she did this for you, and gave us holy music.",
}

return quotes
